### ISSUE TITLE
A mistake in 04_alignment page 9

### ISSUE THREAD

**Issue by Yimin-Xie** [UofABioinformaticsHub/Biotech7005_2016#41]
In the table of this page, I find a mistake.
Because I cannot upload the screenshot, I will use an axis to describe this picture (Korf, 2003).
I set the x axis for the (C, O, E, L, A, C, A, N, T, H), the y axis for the (P, E, L, I, C, A, N), and the values  of (L,E) and (L,L) are incorrect.
The (L,E) should be -2, the (L,L) should be 0.

**Comment by kortschak** [comment]
Yes. This is true. There are a number of errors in the graphics from that book.
